lib/Alchemy/Phrasea/SearchEngine/Elastic/ElasticSearchEngine.php

```diff
@@ -715,6 +714,6 @@ private function buildACLsFilters(array $aclRules)
             $filters[] = $ruleFilter;
         }
 
-        return $filters;
+        return ['bool' => ['should' => $filters]];
```

Comment:
Yes in fact if you have access  granted on bases [1,2] for example the condition
```json
must: [
  {
     "match" : {
         "base_id": 1 
     }
  },
  {
     "match" : {
         "base_id": 2
     }
  },
]
```
Can not be satisfied because record can be in base 1 AND in base 2